testplan/web_ui/testing/src/Report/__tests__/__snapshots__/BatchReport.test.js.snap

```diff
@@ -2229,264 +2229,7 @@ exports[`BatchReport shallow renders the correct HTML structure when report load
-                "entries": Array [
-                  Object {
-                    "category": "testcase",
-                    "description": null,
-                    "entries": Array [
-                      Object {
-                        "category": "DEFAULT",
-                        "description": "passing equality",
-                        "first": 1,
-                        "label": "==",
-                        "line_no": 24,
-                        "machine_time": "2018-10-15T15:30:12.010098+00:00",
-                        "meta_type": "assertion",
-                        "passed": true,
-                        "second": 1,
-                        "type": "Equal",
-                        "utc_time": "2018-10-15T14:30:12.010094+00:00",
-                      },
-                    ],
-                    "logs": Array [],
-                    "name": "test_equality_passing",
-                    "status": "passed",
-                    "status_override": null,
-                    "tags": Object {},
-                    "timer": Object {
-                      "run": Object {
-                        "end": "2018-10-15T14:30:12.132214+00:00",
-                        "start": "2018-10-15T14:30:12.010072+00:00",
-                      },
-                    },
-                    "type": "TestCaseReport",
-                    "uid": "f73bd6ea-d378-437b-a5db-00d9e427f36a",
-                  },
-                ],
-                "logs": Array [],
-                "name": "GammaSuite",
-                "status": "passed",
-                "status_override": null,
-                "tags": Object {},
-                "timer": Object {
-                  "run": Object {
-                    "end": "2018-10-15T14:30:12.158224+00:00",
-                    "start": "2018-10-15T14:30:12.009872+00:00",
-                  },
-                },
-                "type": "TestGroupReport",
-                "uid": "08d4c671-d55d-49d4-96ba-dc654d12be26",
-              },
-            ],
-            "logs": Array [],
-            "name": "Secondary",
-            "status": "passed",
-            "status_override": null,
-            "tags": Object {},
-            "timer": Object {
-              "run": Object {
-                "end": "2018-10-15T14:30:12.159661+00:00",
-                "start": "2018-10-15T14:30:12.009705+00:00",
-              },
-            },
-            "type": "TestGroupReport",
-            "uid": "8c3c7e6b-48e8-40cd-86db-8c8aed2592c8",
-          },
-        ],
-        "meta": Object {},
-        "name": "Sample Testplan",
-        "status": "failed",
-        "status_override": null,
-        "tags_index": Object {},
-        "timer": Object {
-          "run": Object {
-            "end": "2018-10-15T14:30:11.296158+00:00",
-            "start": "2018-10-15T14:30:10.998071+00:00",
-          },
-        },
-        "uid": "520a92e4-325e-4077-93e6-55d7091a3f83",
-      }
-    }
+    report={null}
```

Comment:
That is because now we not simply passing the raw report down, but the filtered one, which get computed if the report is set properly. This testcase just set the state, but not in the right way. I replace it with setState to have it as expected